packages/cactus-plugin-ledger-connector-corda/src/main/typescript/web-services/deploy-contract-jars-endpoint.ts

```diff
@@ -131,14 +132,25 @@ export class DeployContractJarsEndpoint implements IWebServiceEndpoint {
       const body = await this.callInternalContainer(req.body);
       res.status(200);
       res.json(body);
-    } catch (ex) {
+    } catch (ex: unknown) {
+      const stack = LogHelper.getExceptionStack(ex);
+      const messages = LogHelper.getExceptionMessage(ex);
       this.log.error(`${fnTag} failed to serve request`, ex);
-      res.status(500);
-      res.json({
-        error: ex?.message,
-        // FIXME do not include stack trace
-        errorStack: ex?.stack,
-      });
+      if (ex instanceof Error) {
```

Comment:
@Leeyoungone Let's wait for #1381 to be done before we jump into the REST endpoint error handling.

Reply:
Sounds good! So sorry, I've just been plugging away at this epic  -- I can go ahead and wait till the dependent PRs are merged into main. Feel free to ignore the current PRs I have open for endpoint error handling !